Engine/csound_orc.lex

```diff
@@ -130,7 +130,7 @@ SYMBOL          [\[\]+\-*/%\^\?:.,!]
 "|"             { return '|'; }
 "&"             { return '&'; }
 "#"             { return '#'; }
-"�"             { return '~'; } /* \xC2?\xAC */
+"¬"            { return '~'; } /* \xC2?\xAC */
```

Comment:
does that character look right?

Reply:
it's so strange, I've tried to fix it few times, I just can't understand what's up with the encoding, just for sanity check, could you test locally if you can replace it with the original char, because every editor I use create the same diff, and more strange even, is that when I look at your comment, I see this
![2022-12-30 02 29 59](https://user-images.githubusercontent.com/6074754/210025849-1e17166a-7a9f-41a8-83ed-5470b8835970.png)
when I look at the diff I see this strange thing there
![2022-12-30 02 31 36](https://user-images.githubusercontent.com/6074754/210025862-c9533aa7-f92e-4387-a05b-468096d449cf.png)
when I look at the file alone, it again, looks correct
![2022-12-30 02 32 53](https://user-images.githubusercontent.com/6074754/210025924-9aeb9f0a-609f-43eb-9a36-0220316df432.png)

Reply:
![image](https://user-images.githubusercontent.com/2091399/210026484-5947bd82-68ee-45e8-8bcb-431aaa375f48.png)
It looks like the old version is wrong but it didn't look like that first time 😓 I don't really understand. Locally in vscode it looks fine to me. I guess ignore it for now

Reply:
![image](https://user-images.githubusercontent.com/2091399/210026603-b62597b6-04d0-40c0-87e7-d0a1bc71df79.png)
Ah ok, it doesn't render correctly the previous version locally. So the diff looks correct in my screenshot above.

Reply:
could you maybe if you have time, make a PR (or just push to my branch) for the correct character? I wonder if something magical happens when it's correct from your text-editor, afterwards, I will for science, open the file and resave just to see if my editors are still being annoying on this, if that's the case, I'll just not touch this file or dig deeper.

Reply:
I'm unsure of the history here, seems like "~" became an alias for "¬" Oddly, we have a manual entry for "¬" but the example uses ~.

https://csound.com/docs/manual/opbitnot.html

@jpffitch or @vlazzarini : any insight into this? I assume the code should be restored to "¬".